core/src/main/scala/org/ensime/config/ScalariformFormat.scala

```diff
@@ -45,7 +50,11 @@ trait ScalariformFormat {
     ): Sexp = descriptor.preferenceType match {
       case BooleanPreference => value.asInstanceOf[Boolean].toSexp
       case IntegerPreference(_, _) => value.asInstanceOf[Int].toSexp
-      case IntentPreference => value.asInstanceOf[Intent].getClass.getSimpleName.toSexp
```

Comment:
we probably don't need the `asInstanceOf`, can we survive with `.getClass.getSimpleName.toLowerCase.toSexp`?